lib/modulesync.rb

```diff
@@ -201,4 +168,20 @@ def self.update(options)
     end
     exit 1 if errors && options[:fail_on_warnings]
   end
+
+  def self.create_pr_manager
+    github_token = ENV.fetch('GITHUB_TOKEN', '')
+    gitlab_token = ENV.fetch('GITLAB_TOKEN', '')
+
+    if !github_token.empty?
+      require 'modulesync/pr/github'
+      ModuleSync::PR::GitHub.new(github_token, ENV.fetch('GITHUB_BASE_URL', 'https://api.github.com'))
+    elsif !gitlab_token.empty?
+      require 'modulesync/pr/github'
```

Comment:
Should be `gitlab`

Reply:
Fixed via commit 2b5388230fb39dff85e3522db753d20e8c88dfab.